srcpkgs/psuinfo/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'psuinfo'
+pkgname=psuinfo
+version=1.0
+revision=1
+create_wrksrc=yes
+only_for_archs="i686 x86_64"
```

Comment:
Why only x86_64 and i686?

Reply:
No idea on how to test on other architecture. Should I have set `noarch=yes`? I have no experience with packaging for Void.